omap-tutorial/README.md

```diff
@@ -192,7 +195,7 @@ InclusiveCriterion  ClosingPlaybackRoutes  :  Media     Modem     Voice     Tone
 InclusiveCriterion  OpenedCaptureRoutes    :  Media     Modem     Voice     MultiChannel
 InclusiveCriterion  OpenedPlaybackRoutes   :  Media     Modem     Voice     Tones        DeepMedia
 InclusiveCriterion  SelectedInputDevice    :  SubMic             Headset    MainMic    Dmic1    Dmic2     Dmic1
-InclusiveCriterion  SelectedOutputDevice   :  Earpiece   Headset        Ihf
+InclusiveCriterion  SelectedOutputDevices   :  Earpiece   Headset        Ihf
 InclusiveCriterion  AudioSource            :  Default          Mic              None              VoiceCall  VoiceCommunication  VoiceDownlink  VoiceUplink
```

Comment:
I think here I will clean up the AudioCriteria.txt inside the client simulator related to PandaBoardES features as the board does not have all the capability of the SoC enabled. Idea here was more to list all the SoC features for criteria overview.